src/box/lua/schema.lua

```diff
@@ -1735,8 +1735,10 @@ base_index_mt.fselect = function(index, key, opts, fselect_opts)
     local fmt_str = function(x, n)
         if not x then x = '' end
         local str
-        if x:len() <= n then
-            local add = n - x:len()
+        local utf8 = require('utf8')
```

Comment:
I would place all imports (which does not lead to cyclic dependencies) to the head of the file.

If there is a problem with that, it worth to leave a comment in the code.